.vuepress/styles/index.styl

```
@@ -0,0 +1,70 @@
+/**
+ * Custom Styles here.
+ *
+ * ref：https://v1.vuepress.vuejs.org/config/#index-styl
+ */
+
+[data-theme='light']
+ --sideBgColor $sideBgColor
+
+[data-theme='dark']
+ --sideBgColor $sideBgColorDark
+ --bgColor $bgColor
+
+.home .hero img
+  max-width 450px!important
+
+// [data-theme='light']
+//   display none
+
+
+// .home .hero img
+//   max-width 450px!important
+/**
+html
+body
+  background-color var(--bgColor)
+
+.theme-default-content:not(.custom)
+  max-width 1000px
+
+.navbar
+.links
+  background-color var(--navBgColorDark)!important
+
+.navbar
+  filter: drop-shadow(0px 1px 1px var(--bgColor));
+
+*/
+  
+.navbar
+  filter: drop-shadow(0px 1px 1px var(--bgColor));
+
+.sidebar
+  background-color var(--sideBgColor)
+
+div[class*="language-"]
+  pre, pre[class*="language-"]
+    margin-top 0
+
+html {
+  overflow:   scroll;
+}
+::-webkit-scrollbar {
+    width: 0px;
+    background: transparent; /* make scrollbar transparent */
+}
+
+tr
+   &:nth-child(1n)
+     background-color var(--bodyBgColor)
+   &:nth-child(2n)
+     background-color var(--sideBgColor)
+
+	 
+
+.dropdown-wrapper .nav-dropdown
+  border: 1px solid var(--sideBgColor)!important
+  background-color var(--sideBgColor)!important
+  box-shadow: 0px 0px 2px var(--bodyBgColor)
+  right: -12%!important
```

```suggestion
  border 1px solid var(--sideBgColor)!important
  background-color var(--sideBgColor)!important
  box-shadow 0px 0px 2px var(--bodyBgColor)
  right -12%!important
```
Is `!important` needed here?